scripts/src/metrics/metrics.py

```diff
@@ -47,10 +47,10 @@ def get_release_metrics():
             sys.exit(1)
             
         response_json = response.json()
-        if xRateLimit in response_json.headers:
-           print(f'[DEBUG] {xRateLimit} : {response_json.headers[xRateLimit]}')
-        if xRateRemain in response_json.headers:
-            print(f'[DEBUG] {xRateRemain}  : {response_json.headers[xRateRemain]}')
+        if xRateLimit in response_json:
+           print(f'[DEBUG] {xRateLimit} : {response_json[xRateLimit]}')
+        if xRateRemain in response_json:
+            print(f'[DEBUG] {xRateRemain}  : {response_json[xRateRemain]}')
```

Comment:
I don't think this is the correct fix. The mistake in the original was using ```reponse_json.headers``` - it should of been using ```reponse.headers```. New code will not error but will also never produce any output.

Reply:
Changed `response_json[xRateLimit]` to response.headers[xRateLimit]` in the latest commit.